docs/asf.md

```diff
@@ -0,0 +1,721 @@
+would continue without issue if no salaried developers contributed to the
+project.
+
+Example (River):
+> It is expected that Jini development will occur on both salaried time and on
+volunteer time, after hours. While there is reliance on salaried developers
+(currently from Sun, but it's expected that other company's salaried developers
+will also be involved), the Jini Community is very active and things should
+balance out fairly quickly. In the meantime, Sun will support the project in the
+future by dedicating 'work time' to Jini, so that there is a smooth transition.
+
+Example (Wicket):
+> None of the developers rely on Wicket for consulting work, though two -
+Martijn and Eelco - are writing Wicket In Action (publisher Manning) in their
+spare time. Most of the developers use Wicket for their day jobs, some for
+multiple projects, and will do so for a considerable while as their companies
+(specifically Topicus, Cemron and Teachscape) choose Wicket as their development
+framework of choice.
+
+PouchDB:
+
+
+#### Relationships with Other Apache Products:
+
+Apache projects should be open to collaboration with other open source projects
+both within Apache and without. Candidates should be willing to reach outside
+their own little bubbles.
+
+This is an opportunity to talk about existing links. It is also the right place
+to talk about potential future links and plans.
+
+Apache allows different projects to have competing or overlapping goals.
+However, this should mean friendly competition between codebases and cordial
+cooperation between communities.
+
+It is not always obvious whether a candidate is a direct competitor to an
+existing project, an indirect competitor (same problem space, different
+ecological niche) or are just peers with some overlap. In the case of indirect
+competition, it is important that the abstract describes the niche accurately.
+Direct competitors should expect to be asked to summarize architectural
+differences and similarities to existing projects.
+
+Example (OpenJPA):
+> Open JPA will likely be used by Geronimo, requires some Apache products
+(regexp, commons collections, commons lang, commons pool), and supports Apache
+commons logging.
+
+Example (River)
+> Currently the only tie to Apache projects is the starter kit's use of the Ant
+build tool. There are potential future ties (http server, database backend, etc)
+that will be explored.
+
+PouchDB:
+
+
+#### A Excessive Fascination with the Apache Brand:
+
+Concerns have been raised in the past that some projects appear to have been
+proposed just to generate positive publicity for the proposers. This is the
+right place to convince everyone that is not the case.
+
+This is also the right place to build bridges with the community after past
+misdemeanors (for example, if any of those associated with the proposal have -
+in the past - sort to associate themselves with the Apache brand in factually
```

Comment:
```suggestion
in the past - sought to associate themselves with the Apache brand in factually
```
Looks like the source is https://cwiki.apache.org/confluence/display/INCUBATOR/New+Podling+Proposal but I don't have a login to correct there.